src/api/exchange/ExchangeApi.ts

```diff
@@ -109,6 +121,38 @@ export class ExchangeApiImpl extends DepositApiImpl implements ExchangeApi {
     return decodeAuctionElements(encodedOrders)
   }
 
+  public async getOrdersPaginated({
+    userAddress,
+    networkId,
+    offset,
+    pageSize,
+  }: GetOrdersPaginatedParams): Promise<GetOrdersPaginatedResult> {
+    const contract = await this._getContract(networkId)
+    const _pageSize = pageSize || DEFAULT_ORDERS_PAGE_SIZE
```

Comment:
could also set the default in the params above as `pageSize = DEFAULT_ORDERS_PAGE_SIZE`